src/components/sections/CTAFull.astro

```diff
@@ -5,15 +5,42 @@ interface Props {
     body?: string;
     button?: string;
     href?: string;
+    fullHeight?: boolean;
+    background?:
+        | {
+              from: string;
+              to: string;
+          }
```

Comment:
In future, consider create a interface for readability types

Reply:
Ok i'll consider that